### PR TITLE
Only update tm_user.last_authenticated once per minute

### PR DESCRIPTION
**Pull request by rawlinp**
This is to avoid row-locking when the same user logs in frequently.

<hr/>

## Which Traffic Control components are affected by this PR?

- Traffic Ops

## What is the best way to verify this PR?

As the same user, log in more than once a minute while checking the user's `last_authenticated` time. It should only update once per minute.

## PR submission checklist
- [x] ~This PR has tests~ 
- [x] ~This PR has documentation~ 
- [x] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

File: traffic_ops/traffic_ops_golang/login/login.go [hunk 1]
```diff
@@ -63,7 +63,9 @@ WHERE name='tm.instance_name' AND
 `
 const userQueryByEmail = `SELECT EXISTS(SELECT * FROM tm_user WHERE email=$1)`
 const setTokenQuery = `UPDATE tm_user SET token=$1 WHERE email=$2`
-const UpdateLoginTimeQuery = `UPDATE tm_user SET last_authenticated = now() WHERE username=$1`
+
+// UpdateLoginTimeQuery is meant to only update the last_authenticated field once per minute in order to avoid row-locking when the same user logs in frequently.
+const UpdateLoginTimeQuery = `UPDATE tm_user SET last_authenticated = now() WHERE username=$1 AND last_authenticated < now() - INTERVAL '1 minute'`
```
Comment (zrhoffman) [inline traffic_ops/traffic_ops_golang/login/login.go:68]: Since the other SQL keywords are uppercase, should `now()` be uppercase? Should `minute`?

Comment (zrhoffman) [inline traffic_ops/traffic_ops_golang/login/login.go:68]: If `last_authenticated` is null, this condition will be false

Comment (rawlinp) [reply traffic_ops/traffic_ops_golang/login/login.go:68]: Yeah, just found that too, will fix 👍

**Review by zrhoffman — APPROVED** [review]
Looks good! This def should help performance.

**Pull request merged**